tools/mininet/Dockerfile

```diff
@@ -53,24 +53,19 @@ RUN cp --parents /usr/local/lib/libbmpi.so.0 /output
 
 # As well as the P4Runtime Python bindings installed by PI. This is not needed
 # to run mininet, but it's useful to execute Python scripts acting as a
-# P4Runtime client, e.g. control plane apps or PTF tests.
-RUN bazel build @com_github_p4lang_p4runtime//:p4runtime_py_grpc
-
+# P4Runtime client, e.g. control plane apps or PTF tests. Also includes
+# Google API (e.g. status proto) needed by P4Runtime.
 ENV PYTHON_PACKAGE_BASE /output/usr/local/lib/python2.7/dist-packages
-RUN mkdir -p $PYTHON_PACKAGE_BASE/p4 && \
-    cp -r bazel-bin/external/com_github_p4lang_p4runtime/p4/* $PYTHON_PACKAGE_BASE/p4 && \
+COPY ./py_out /p4runtime
```

Comment:
I would add a comment explaining what `./py_out` is and how it should be provided. Also, since the build of this image now depends on the execution of some other steps first (to generate `py_out`), the instructions at the beginning of this file should be updated. You could simply say "Look at .circleci/config.yaml for an example of commands to build this image."